packages/dynamite/dynamite_runtime/lib/src/dynamite_client.dart

```diff
@@ -425,33 +424,41 @@ class DynamiteClient {
     return executeRawRequest(
       method,
       uri,
-      {...?baseHeaders, ...headers},
+      {
+        ...?baseHeaders,
+        ...headers,
```

Comment:
Should we change the order to ensure the base headers are not overridden or do we consider it as a feature?

Reply:
I think that a header specified in the specs should have higher priority.
If an operation has a `useragent` header parameter it should be used over the client one.

Reply:
Sounds good.